cli/commands/authenticate.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const authenticationTasks = require('../../lib/authentication-tasks');
+const config = require('../../config');
+const logger = config.logger;
+
+module.exports = function(program) {
+	program.command('authenticate')
+		.description('Obtain AWS session credentials and set them as environment variables.')
+		.option('-p, --profile <profile>', 'Profile to use to obtain session credentials')
```

Comment:
Perhaps add documentation to indicate this profile must have already been defined in your `~/.aws/credentials` file or using the `aws configure` CLI command.